assets/src/scripts/monitoring-location/components/hydrograph/method-picker.js

```diff
@@ -28,37 +34,57 @@ const updateAvailableMethods = function(selectElem, methods, store) {
     }
 
     methods.forEach((method) => {
-        selectElem.append('option')
-            .text(method.methodDescription ? `${method.methodDescription}` : 'None')
+        const hasPointsInTimeSpan = method.pointCount > 0;
+        container.append('option')
+            .text(`${method.methodDescription}`)
+            .attr('class', 'method-option sampling-method-selection')
             .attr('selected', method.methodID === selectedMethodID ? true : null)
-            .node().value = method.methodID;
+            .attr('value', method.methodID)
+            .property('disabled', !hasPointsInTimeSpan ? true : null);
         });
 };
 
-/*
+/**
  * Draw the method picker. It will be set initially to the preferred method id if not already
  * set to a specific, available method id. The picker will be hidden if only one method
  * is available for the IV data.
- * @param {D3 selection} elem
- * @param {Redux store} store
+ * @param {D3 selection} container - An HTML element to serve as the base for attaching other elements.
+ * @param {String} parameterCode - The five digit USGS that is unique for each parameter.
+ * @param {Redux store} store - A complex JavaScript object that maps the current state of the application.
  */
-export const drawMethodPicker = function(elem, store) {
-    const pickerContainer = elem.append('div')
+export const drawMethodPicker = function(container, parameterCode, store) {
+    const gridRowSamplingMethodSelection = container.append('div')
+        .attr('id', 'primary-sampling-method-row')
+        .attr('class', 'grid-container grid-row-inner sampling-method-selection');
+    const methodSelectionContainer = gridRowSamplingMethodSelection.append('div')
+        .attr('id', `method-selection-container-${parameterCode}`)
```

Comment:
Is there a reason we need this id? I think we should try not to put ids or classes on things unless needed. It just increases the cognitive load. In the above example for instance, you have the class "sampling-method-selection". That class is used for divs (at different levels), options, and labels.  I see that you are using it in parameters.js but I don't think it needs to be on all of these things (probably just the "option" tag)

Reply:
Good eye. Yes, some of this is left over because I was having a devil of a time getting the '.remove()' to work. I'll see if I can clean this up.

Reply:
You might want to put a clarifying comment on the spot where you do the exclude.

Reply:
Can do!